src/constants/script_constants.asm

```diff
@@ -0,0 +1,147 @@
+	const EVENT_FLAG_43                         ; $43
+	const EVENT_FLAG_44                         ; $44
+	const EVENT_FLAG_45                         ; $45
+	const EVENT_FLAG_46                         ; $46
+	const EVENT_FLAG_47                         ; $47
+	const EVENT_FLAG_48                         ; $48
+	const EVENT_FLAG_49                         ; $49
+	const EVENT_FLAG_4A                         ; $4a
+	const EVENT_FLAG_4B                         ; $4b
+	const EVENT_FLAG_4C                         ; $4c
+	const EVENT_FLAG_4D                         ; $4d
+	const EVENT_FLAG_4E                         ; $4e
+	const EVENT_FLAG_4F                         ; $4f
+	const EVENT_FLAG_50                         ; $50
+	const EVENT_FLAG_51                         ; $51
+	const EVENT_FLAG_52                         ; $52
+	const EVENT_FLAG_53                         ; $53
+	const EVENT_FLAG_54                         ; $54
+	const EVENT_FLAG_55                         ; $55
+	const EVENT_FLAG_56                         ; $56
+	const EVENT_FLAG_57                         ; $57
+	const EVENT_FLAG_58                         ; $58
+	const EVENT_FLAG_59                         ; $59
+	const EVENT_FLAG_5A                         ; $5a
+	const EVENT_FLAG_5B                         ; $5b
+	const EVENT_FLAG_5C                         ; $5c
+	const EVENT_FLAG_5D                         ; $5d
+	const EVENT_FLAG_5E                         ; $5e
+	const EVENT_FLAG_5F                         ; $5f
+	const EVENT_FLAG_60                         ; $60
+	const EVENT_FLAG_61                         ; $61
+	const EVENT_FLAG_62                         ; $62
+	const EVENT_FLAG_63                         ; $63
+	const EVENT_FLAG_64                         ; $64
+	const EVENT_FLAG_65                         ; $65
+	const EVENT_FLAG_66                         ; $66
+	const EVENT_FLAG_67                         ; $67
+	const EVENT_FLAG_68                         ; $68
+	const EVENT_FLAG_69                         ; $69
+	const EVENT_FLAG_6A                         ; $6a
+	const EVENT_FLAG_6B                         ; $6b
+	const EVENT_FLAG_6C                         ; $6c
+	const EVENT_FLAG_6D                         ; $6d
+	const EVENT_FLAG_6E                         ; $6e
+	const EVENT_FLAG_6F                         ; $6f
+	const EVENT_FLAG_70                         ; $70
+	const EVENT_FLAG_71                         ; $71
+	const EVENT_FLAG_72                         ; $72
+	const EVENT_FLAG_73                         ; $73
+	const EVENT_FLAG_74                         ; $74
+	const EVENT_FLAG_75                         ; $75
+	const EVENT_FLAG_76                         ; $76
+EVENT_FLAG_AMOUNT EQU const_value
+
+EVENT_FLAG_BYTES EQU $40
+
+; EVENT_IMAKUNI_STATE
+; Starts at 0, Talking to lass moves it to MENTIONED (1), then
+; talking to Imakuni at least once sets it to TALKED (2)
+IMAKUNI_MENTIONED   EQU 1
+IMAKUNI_TALKED      EQU 2
```

Comment:
So "event flags" are larger than 1 bit?

Reply:
they are of arbitrary length, many are 1 bit though